scripts/categorizer.py

```diff
@@ -1,5 +1,8 @@
 import collections
 import re
+import hashlib
```

Comment:
you are importing a lot libraries and this is getting kind of long, consider using vulture to make sure all variables are used, maybe labeling if any of these are third party libraries or maybe instead make a requirements.txt to capture anything that is 3rd party. Those both would accomplish giving people using the code a quick way to know whether they need to install anything. Also add requriements.txt above.

Reply:
Everything comes standard with python.